ethcore/wasm/src/runtime.rs

```diff
@@ -626,16 +627,30 @@ impl<'a, 'b> Runtime<'a, 'b> {
 	fn user_panic(&mut self, context: InterpreterCallerContext)
 		-> Result<Option<interpreter::RuntimeValue>, InterpreterError>
 	{
-		let msg_len = context.value_stack.pop_as::<i32>()? as u32;
-		let msg_ptr = context.value_stack.pop_as::<i32>()? as u32;
-
-		let msg = String::from_utf8(self.memory.get(msg_ptr, msg_len as usize)?)
-			.map_err(|_| UserTrap::BadUtf8)?;
-
-		trace!(target: "wasm", "Contract custom panic message: {}", msg);
-
-		Err(UserTrap::Panic(msg).into())
-	}
+        let payload_len = context.value_stack.pop_as::<i32>()? as u32;
```

Comment:
use tabs instead of spaces

Reply:
Thanks! It happened after I copied this chunk after rustfmt : (